sdk/program/src/stake/state.rs

```diff
@@ -305,7 +305,7 @@ impl Authorized {
         signers: &HashSet<Pubkey>,
         new_authorized: &Pubkey,
         stake_authorize: StakeAuthorize,
-        lockup_custodian_args: Option<(&Lockup, &Clock, Option<&Pubkey>)>,
+        lockup_custodian_args: (&Lockup, &Clock, Option<&Pubkey>),
```

Comment:
Good point, I didn't notice that. I don't mind reverting it.. though I don't think this method should have been exposed publicly in the first place since it's an implementation detail of the program.

Reply:
I reverted the interface change

Reply:
Yeah it's definitely not meant to be exposed. Considering the change didn't break SPL, I think it would be a safe change to make, since it's arguably more correct to pass the parameter always, but I also don't feel too strongly about it.

Reply:
Yeah that's exactly how I feel. If you feel like un-exposing the interface, I'd be happy to review. But I won't likely be doing it myself